bHY6yh-d24602/cod-bxxqqq_bHY6yh-d24602.xml

```diff
@@ -0,0 +1,169 @@
+          <lb ed="#B"/>neque eum qui bene mereri para<lb ed="#B" break="no"/>tus 
+          fuerat prodere uoluit 
+          <lb ed="#B"/>¶ Cassus carbonis nomen infesto
+          <lb ed="#B"/>animo ut pote inimicissimum sibi 
+          <lb ed="#B"/>detulerat sed tamen scrinium eius 
+          <lb ed="#B"/>a seruo carbonis allatum ad se et 
+          <lb ed="#B"/>plura continens quibus facile 
+          <lb ed="#B"/>opprimi posset ut erat signa<lb ed="#B" break="no"/>tum 
+          cum seruo catenato ad eum remi<lb ed="#B" break="no"/>sit</quote>
+        </p>
+        <p xml:id="bHY6yh-d24602-d1e194"> /
+          ¶ <ref xml:id="bHY6yh-d24602-Rd1e195">libro 9 titulo</ref> <quote xml:id="bHY6yh-d24602-Qd1e196">do perfidia filiam vir<lb ed="#B" break="no"/>ginem 
+          spiriturii tarptii qui arci pre<lb ed="#B" break="no"/>erat 
+          Romolo regnante aquam 
+          <lb ed="#B"/>sacris petitum extra menia egressam 
+          <lb ed="#B"/>tacius ut armatos sabinos in arcem 
+          <lb ed="#B"/>secutum reciperet corrupit mercedis no<lb ed="#B" break="no"/>mine 
+          pactam que in sinistris manibus 
+          <lb ed="#B"/>gerebant erant autem hiis armille 
+          <lb ed="#B"/>et anuli magno ex pondere auri 
+          <lb ed="#B"/>loco potitum agrum sabinorum pu<lb ed="#B" break="no"/>ellam 
+          premium flagitantem armis 
+          <lb ed="#B"/>obrutam necauit perinde quasi per<lb ed="#B" break="no"/>missum 
+          quod ea quoque leuis gesta<lb ed="#B" break="no"/>uerat 
+          soluit Absit reprehensio 
+          <lb ed="#B"/>quia impia proditio celeri pena vin<lb ed="#B" break="no"/>dicata 
+          est</quote>
+        </p>
+        <p xml:id="bHY6yh-d24602-d1e226">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d24602-Rd1e231">AMbrosius<!--line jumps up--> de officiis libro 3 et finale<!--line jumps down--> capitulo 16 et finale</ref> <quote xml:id="bHY6yh-d24602-Qd1e230">Inimicus 
+          <lb ed="#B"/>vitari potest amicus non potest 
+          <lb ed="#B"/>si insidiari uelit</quote>
+        </p>
+        <p xml:id="bHY6yh-d24602-d1e238">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d24602-Rd1e244">IAcobus beneuen<!--line jumps up-->tanus capitulo 9</ref> <quote xml:id="bHY6yh-d24602-Qd1e242">Hoc<!--line jumps down--> ego te moneo 
+          <lb ed="#B"/>fugias ut proditionem Turpius 
+          <lb ed="#B"/>hoc enim crimine crimen ad est Pro<lb ed="#B" break="no"/>ditor 
+          et nullus tibi coniungatur 
+          <lb ed="#B"/>amore Sepe ab amicicie mors 
+          <lb ed="#B"/>inimica datur</quote>
+        </p>
+        <p xml:id="bHY6yh-d24602-d1e254">
+          <lb ed="#B"/><ref xml:id="bHY6yh-d24602-Rd1e262">Avctor libri<!--line jumps up--> qui incipit as<!--line jummps down-->trolabii</ref> <quote xml:id="bHY6yh-d24602-Qd1e258">Proditor 
+          <lb ed="#B"/>alterius non tibi fidus erit
+          . Obsequi superant meretrix et pro<lb ed="#B" break="no"/>ditor 
```

Comment:
was this supposed to be a line break that accidentally became a `.`. Can you confirm?
```suggestion
          <lb ed="#B"/>Obsequi superant meretrix et pro<lb ed="#B" break="no"/>ditor 
```